dashboard/db/migrate/20180601174113_create_shared_blockly_functions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSharedBlocklyFunctions < ActiveRecord::Migration[5.0]
+  def change
+    create_table :shared_blockly_functions do |t|
```

Comment:
I would specify the `level_type` for which these functions will be included, but we can always add that later if/when we actually want to use it elsewhere.

Reply:
Good call, added in d987195cf94c8bc14f19e3260d53233b264b166f.